### ISSUE TITLE
Could you please provide the training set you constructed?

### ISSUE THREAD

**Issue by Kausal-Lei** [google-research-datasets/TimeDial#1]
I've tried to construct the training set with the method you introduced in this paper. But my accuracy on the test set is really low,  I want to figure out whether it's caused by the training set I made. I'll be really appreciated if you can provide the training set you constructed.

**Comment by qkaren** [comment]
Thanks for your interest in your dataset! Please check the link below for the entire dataset.
[https://drive.google.com/file/d/18szdnYG2pi4o6eesT9-gBWya0NohLQoV/view?usp=sharing]()

**Comment by Kausal-Lei** [comment]
> Thanks for your interest in your dataset! Please check the link below for the entire dataset. https://drive.google.com/file/d/18szdnYG2pi4o6eesT9-gBWya0NohLQoV/view?usp=sharing

Thank you very much!
